quality_modifiers.rb

```diff
@@ -125,7 +125,7 @@ def modifiers
       }),
 
       Modifier.new("Empty README", "The README is the front page of a library. To have this applied you may have a very empty README.", -8, Proc.new { |spec, cd_stats, stats, cp_stats, owners|
-        cd_stats[:readme_complexity].to_i < 25 || spec.documentation_url != nil
+        cd_stats[:readme_complexity].to_i < 25 && spec.documentation_url != nil
```

Comment:
Hmm, so if I have a good README and I have a proper documentation site. I get punished and I lose 8 points for having an "empty" readme?

I think the check for documentation_url should be removed completely, it has nothing to do with a README.